samples/snippets/src/main/java/com/example/dialogflow/ConversationProfileManagement.java

```diff
@@ -0,0 +1,137 @@
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.dialogflow;
+
+// [START dialogflow_create_conversation_profile_article_faq]
+
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.dialogflow.v2.ConversationProfile;
+import com.google.cloud.dialogflow.v2.ConversationProfilesClient;
+import com.google.cloud.dialogflow.v2.CreateConversationProfileRequest;
+import com.google.cloud.dialogflow.v2.HumanAgentAssistantConfig;
+import com.google.cloud.dialogflow.v2.HumanAgentAssistantConfig.SuggestionConfig;
+import com.google.cloud.dialogflow.v2.HumanAgentAssistantConfig.SuggestionFeatureConfig;
+import com.google.cloud.dialogflow.v2.HumanAgentAssistantConfig.SuggestionQueryConfig;
+import com.google.cloud.dialogflow.v2.HumanAgentAssistantConfig.SuggestionQueryConfig.KnowledgeBaseQuerySource;
+import com.google.cloud.dialogflow.v2.HumanAgentAssistantConfig.SuggestionTriggerSettings;
+import com.google.cloud.dialogflow.v2.KnowledgeBaseName;
+import com.google.cloud.dialogflow.v2.LocationName;
+import com.google.cloud.dialogflow.v2.SuggestionFeature;
+import com.google.cloud.dialogflow.v2.SuggestionFeature.Type;
+import java.io.IOException;
+import java.util.Optional;
+
+public class ConversationProfileManagement {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "my-location";
+    String conversationProfileDisplayName = "my-conversation-profile-display-name";
+
+    // Create a conversation profile
+    String articleSuggestionKnowledgeBaseId = "my-fake-article-suggestion-knowledge-base-id";
+    String faqKnowledgeBaseId = "my-fake-faq-knowledge-base-id";
+    createConversationProfileArticleFaq(
+        projectId,
+        conversationProfileDisplayName,
+        location,
+        Optional.of(articleSuggestionKnowledgeBaseId),
+        Optional.of(faqKnowledgeBaseId));
+  }
+
+  // Create a conversation profile with given values about article suggestion or faq
+  public static ConversationProfile createConversationProfileArticleFaq(
+      String projectId,
+      String displayName,
+      String location,
+      Optional<String> articleSuggestionKnowledgeBaseId,
+      Optional<String> faqKnowledgeBaseId)
+      throws ApiException, IOException {
+    try (ConversationProfilesClient conversationProfilesClient =
```

Comment:
whitespace before `try`?

Reply:
What line width are yous set to?  I think we are at least 100 for Java, and maybe 120 for devrel.

Reply:
100 for Java, it is Google's recommendation right?